apps/aevm/src/aevm_data.erl

```diff
@@ -148,6 +148,11 @@ convert(_, _, MaxSize, _, _Visited, string, Val, Heap, BaseAddr) ->
     Bytes = Words * 32,
     check_size(MaxSize, Bytes),
     {BaseAddr, {aeb_heap:maps_with_next_id(Heap), Bytes, [get_chunk(Heap, Val, Bytes)]}};
+convert(_, _, _, _, _, {bytes, Len}, Val, Heap, _) when Len =< 32 ->
```

Comment:
I see type of this `{bytes, _}` is specified to be `aeb_aevm_data:type()`.

Reply:
Gah, I think one commit was lost somewhere in the rebasing, this I did remove last night 😞

Reply:
Based on rebar.lock, type is defined at https://github.com/aeternity/aebytecode/blob/1526ad3bf057e72a1714aea0430b001bd1d576c9/src/aeb_aevm_data.erl#L14-L27

Reply:
We did revert the addition of `bytes` at the VM-level

Reply:
I do not see bytes 🤔 
```
-type data() :: none
              | {some,   data()}
              | {option, data()}
              | word
              | string
              | {list,   data()}
              | {tuple, [data()]}
              | {variant, integer(), [data()]}
              | integer()
              | binary()
              | [data()]
              | {}
              | {data()}
              | {data(), data()}.
```